tests/ui/unwrap_or.rs

```diff
@@ -0,0 +1,5 @@
+#![warn(clippy)]
+
+fn main() {
+    let s = Some(String::from("test string")).unwrap_or("Fail".to_string()).len();
```

Comment:
can you create a test with the newlines like in the issue?

Reply:
Ah its newlines you want tested? - I thought it was just a method call after the part to replace. I'll add a new test.

Reply:
You can add it to the same test